### ISSUE TITLE
'System.IO.Unsafe' does not export 'unsafeDupableperformIO' failure during install

### ISSUE THREAD

**Issue by danielbprice** [meiersi/blaze-builder#12]
I'm a Haskell uber-novice; my interactions with this package are in service of trying to get 'pandoc' to install, which used to work fine.  Now, however, they are failing because blaze-builder is failing to install.  Here is what the error looks like:

```
Resolving dependencies...
Configuring blaze-builder-0.3.2.0...
Building blaze-builder-0.3.2.0...
Preprocessing library blaze-builder-0.3.2.0...
[ 1 of 13] Compiling Blaze.ByteString.Builder.Internal.Types ( Blaze/ByteString/Builder/Internal/Types.hs, dist/build/Blaze/ByteString/Builder/Internal/Types.o )
[ 2 of 13] Compiling Blaze.ByteString.Builder.Internal.Write ( Blaze/ByteString/Builder/Internal/Write.hs, dist/build/Blaze/ByteString/Builder/Internal/Write.o )
[ 3 of 13] Compiling Blaze.ByteString.Builder.Internal.Buffer ( Blaze/ByteString/Builder/Internal/Buffer.hs, dist/build/Blaze/ByteString/Builder/Internal/Buffer.o )
[ 4 of 13] Compiling Blaze.ByteString.Builder.Internal.UncheckedShifts ( Blaze/ByteString/Builder/Internal/UncheckedShifts.hs, dist/build/Blaze/ByteString/Builder/Internal/UncheckedShifts.o )
[ 5 of 13] Compiling Blaze.ByteString.Builder.Internal ( Blaze/ByteString/Builder/Internal.hs, dist/build/Blaze/ByteString/Builder/Internal.o )

Blaze/ByteString/Builder/Internal.hs:66:26:
    Module `System.IO.Unsafe' does not export `unsafeDupablePerformIO'
Failed to install blaze-builder-0.3.2.0
cabal: Error: some packages failed to install:
blaze-builder-0.3.2.0 failed during the building phase. The exception was:
ExitFailure 1
blaze-html-0.6.1.1 depends on blaze-builder-0.3.2.0 which failed to install.
blaze-markup-0.5.1.5 depends on blaze-builder-0.3.2.0 which failed to install.
highlighting-kate-0.5.5 depends on blaze-builder-0.3.2.0 which failed to
install.
pandoc-1.11.1 depends on blaze-builder-0.3.2.0 which failed to install.
```

```
$ ghc --version
The Glorious Glasgow Haskell Compilation System, version 7.0.2
```

By dumb experimentation, I found that if I manually install blaze-builder 0.3.1.0, my pandoc install subsequently works.

**Comment by danielbprice** [comment]
I believe this also breaks with the ghc available as part of Centos 6.4.

**Comment by meiersi** [comment]
Hi @danielbprice , thanks for the report. This is likely a bug in one of the macros that we're using to handle the changes in the base library of different version of GHC. I don't have quick access to a GHC 7.0.x. This is indeed a rather old version (more than two years). However, I'll see what I can do about this bug.

In the meantime, I recommend installing the newest Haskell platform (http://www.haskell.org/platform/), which should solve your problems with installing `pandoc`.

**Comment by danielbprice** [comment]
Sorry, accidentally clicked "close" apparently.

**Comment by itchyny** [comment]
I have encountered this problem while testing my codes with multiple versions of GHC on travis-ci.
I think the following issue will help you fix.
https://github.com/vincenthz/hs-cipher-rc4/issues/1

**Comment by meiersi** [comment]
Closed via #21
